player/lua/stats.lua

```diff
@@ -297,9 +297,10 @@ end
 -- exclude: Optional table containing keys which are considered invalid values
 --          for this property. Specifying this will replace empty string as
 --          default invalid value (nil is always invalid).
-local function append_property(s, prop, attr, excluded)
+-- cached : If true, use get_property_cached instead of get_property_osd
+local function append_property(s, prop, attr, excluded, cached)
     excluded = excluded or {[""] = true}
-    local ret = mp.get_property_osd(prop)
+    local ret = cached and get_property_cached(prop) or mp.get_property_osd(prop)
```

Comment:
hmm, does this work? If `get_property_cached` returns falsely, it would fall-through to `get_property_osd`, which is not what we want, no?